website/blog/2024-10-23-release-0.76-new-architecture.md

```diff
@@ -0,0 +1,222 @@
+---
+title: 'React Native 0.76 - New Architecture by default, React Native DevTools, Box Shadows and more'
+authors: [blakef, cipolleschi, frankcalise, gabrieldonadel]
+tags: [engineering]
+date: 2024-10-23
+---
+
+Today we are excited to release React Native 0.76! This is a major milestone for React Native, with the New Architecture enabled by default, and the new React Native DevTools. This has been the culmination of 6 years of hard work from our incredible community of developers to make this day happen.
+
+### Highlights
+
+- [React Native New Architecture by default](#react-native-new-architecture-by-default)
+- [React Native DevTools](#react-native-devtools)
+- [Box Shadow and Filter style props](#box-shadow-and-filter-style-props)
+
+### Breaking Changes
+
+- [Removed the dependency on the react-native-community/cli](#removed-the-dependency-on-the-react-native-communitycli)
+- [[Android] React Native C++ code is distributed as libreactnative.so](#android-react-native-is-distributed-as-a-single-library)
+- [Updates to Minimum iOS and Android SDK requirements](#updates-to-minimum-ios-and-android-sdk-requirements)
+
+<!--truncate-->
+
+## Highlights
+
+### React Native New Architecture by default
+
+The New Architecture is finally here and starting from 0.76 is enabled by default in your projects.
+
+Thanks to our collaboration with the whole community, we feel now confident to make this step. We verified that more than [880 libraries](https://reactnative.directory/?newArchitecture=true) are compatible with the New Architecture, including all the Expo Modules.
```

Comment:
> Thanks to our collaboration with the whole community

While we were part of the collaboration, this tone gives off some self-importance.

Just give the community the credit:

> Thanks to the community's validation and testing, we now feel confident making this change.